.circleci/config.yml

```diff
@@ -653,3 +641,23 @@ commands:
             gpgv codecov.SHA256SUM.sig codecov.SHA256SUM
             shasum -a 256 -c codecov.SHA256SUM
             chmod +x codecov
+  setup_test_results:
+    steps:
+      - run:
+          name: Setup test results
+          command: |
+            mkdir -p ~/test-results/junit/
+            find . -type f -regex ".*/target/.*-reports/TEST-.*xml" -exec cp {} ~/test-results/junit/ \;
```

Comment:
This excludes the "failsafe-summary.xml" that used to cause uploading the tests to fail (since CircleCI couldn't parse the xml). The total test count doesn't seem to be affected so maybe this is okay (unless the fail safe summary is needed elsewhere)? I suppose the summary could also be uploaded as an artifact if it's needed.

Reply:
I don't think the fail safe summary is useful if circle ci cannot use it. 
If the total test count remains the same, I think we're good